internal/processors/cloud-vendor-aggregator/azure/client.go

```diff
@@ -0,0 +1,87 @@
+// Copyright Mia srl
+// SPDX-License-Identifier: Apache-2.0
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package azure
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+
+	"github.com/mia-platform/integration-connector-agent/internal/azure"
+	"github.com/mia-platform/integration-connector-agent/internal/processors/cloud-vendor-aggregator/commons"
+)
+
+var (
+	apiVersionsMap = map[string]string{
```

Comment:
why are there two api versions map? This and line 77

Reply:
My bad, is a relic of an alternative rework that has been reworked in this one, I'm removing it.